src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,317 @@
+/* eslint-disable @typescript-eslint/prefer-optional-chain */
```

Comment:
You can remove this eslint-disable and correct line 154, see comment below.

Reply:
do it